specification/eventgrid/data-plane/Microsoft.ContainerRegistry/stable/2018-01-01/ContainerRegistry.json

```diff
@@ -25,6 +25,24 @@
             ],
             "properties": {}
         },
+        "ContainerRegistryChartPushedEventData": {
```

Comment:
>Chart [](start = 26, length = 5)

referring to the other comment.. should this also be renamed to Artifact ?

Reply:
`Chart` is one type of `Artifact` and there will be other event types for other event types that inherit the same `ContainerRegistryArtifactEventData`

Reply:
sounds good.

---
In reply to: [252465044](https://github.com/Azure/azure-rest-api-specs/pull/5109#discussion_r252465044) [](ancestors = 252465044)